Tests/DatadogTests/Datadog/RUM/RUMFeatureTests.swift

```diff
@@ -196,7 +198,7 @@ class RUMFeatureTests: XCTestCase {
         // Given
         try RUMMonitor.initialize(in: core, configuration: .mockAny())
 
-        core.scope(for: DatadogRUMFeature.name)?.eventWriteContext { _, writer in
+        core.scope(for: "rum")?.eventWriteContext { _, writer in
```

Comment:
It's a test, so it's not a big deal, but I'm wondering why we lost access to the feature's name 🤔

Reply:
We didn't actually. I just need to do `@testable import DatadogRUM` 👍